eZ/Publish/API/Repository/Tests/ObjectStateServiceTest.php

```diff
@@ -229,11 +229,11 @@ public function testCreateObjectStateGroup()
         $objectStateGroupCreate->defaultLanguageCode = 'eng-US';
         $objectStateGroupCreate->names = array(
             'eng-US' => 'Publishing',
-            'eng-GB' => 'Sindelfingen',
+            'ger-DE' => 'Sindelfingen',
```

Comment:
Note: this is just for unification of names and descriptions to use common data provider for multi-language / unified logic.